services/integration/src/controllers/project.js

```diff
@@ -0,0 +1,60 @@
+const Project = require('../model/projectModel');
+
+const { asyncHandler } = require('../utils');
+//routes
+const test = (req, res) => {
+    const { id } = req.params;
+    const { active } = req.query;
+    res.send({ message: id, active });
+};
+
+const createProject = asyncHandler(async(req, res) => {
+    let project = new Project(req.body);
+   await project.save(function (err) {
+        if (err) {
+            return next(err);
+        }
+        res.send('Project Created successfully')
+    })
+
+});
+
+const getProject = asyncHandler(async (req, res) => {
```

Comment:
You are inside a file named as `project`. Having function with the word `project` in its names is unnecessary . Please, start to name this kind of function as just `create`,  `get`, `find`(instead of `getProjects`), and so on...

Reply:
so , if I , call to list proyect , but I want to distinguish of list proyects? how do I do ?

Reply:
well , I do , the changes , and soon I show you